include/boost/container/container_fwd.hpp

```diff
@@ -115,7 +115,8 @@ template <class T, std::size_t Capacity>
 class static_vector;
 
 template < class T, std::size_t N
-         , class Allocator= new_allocator<T> >
+         , class Allocator= new_allocator<T>
+         , std::size_t alignment= 0 >
```

Comment:
If anything, the default value for the alignment value may as well be 1, indicating the minimum alignment value.